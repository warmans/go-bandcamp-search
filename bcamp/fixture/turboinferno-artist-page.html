


<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html   xmlns:og="http://opengraphprotocol.org/schema/"
        xmlns:fb="http://www.facebook.com/2008/fbml">

<head>
    <title>Music | Turbo Inferno</title>





    <meta name="Description" content="
Turbo Inferno
Bastard Noise desde la rendija de una puerta mal cerrada en la cabina de control de una estacion de metro.
">



    <link rel="shortcut icon" href="https://s4.bcbits.com/img/bc_favicon.ico">
    <link rel="apple-touch-icon" href="https://s4.bcbits.com/img/bc_favicon.ico">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <!--<meta name="apple-mobile-web-app-status-bar-style" content="black translucent">-->

    <meta name="norton-safeweb-site-verification" content="ejd7gu5uojnhaq1f4mkwkx8rbydsxtne25pyxbyv24x7e2b-voeqw8wohh9j183bfyeq109vyl4vd396-wkx1gog2vv4g21x39voxpe75a96ahp98-nookkmva7n5t7a" />



    <meta name="title" content="Turbo Inferno">
    <meta property="og:title" content="Turbo Inferno">
    <meta property="og:type" content="band">
    <meta property="og:url" content="http://turboinferno.bandcamp.com">
    <meta property="og:site_name" content="Turbo Inferno">
    <meta property="fb:app_id" content="165661066782720">

    <meta property="og:description"  content="Bastard Noise desde la rendija de una puerta mal cerrada en la cabina de control de una estacion de metro.">













    <link type="text/css" rel="stylesheet" href="https://s4.bcbits.com/tmpdata/cache/global_bundle_5b7e3e59ca16f954d51f40a64e02edd0.css">
    <link type="text/css" rel="stylesheet" href="https://s4.bcbits.com/tmpdata/cache/tralbum_bundle_031b212f0e81aa9d4c9b3519b491bdd9.css">



    <script type="text/javascript">
        var isDebug = false;
        var ErrorCollector = {"enabled":true,"sampleRate":null};
        var CfgShare2 = true; // fixme: remove after Share2 always on
    </script>




    <script type="text/javascript">

    var isDebug = null;
    var siteroot = "http://bandcamp.com";
    var siteroot_https = "https://bandcamp.com";
    var siteroot_current = "https://bandcamp.com";
    var band_url = "http://turboinferno.bandcamp.com";
    var isProduction = true;


    document.documentElement.className += " js";


    var Browser = {"mobile_app_compatible":false,"type":"webkit","platform_closed":false,"make":"chrome","version":[48,0,2564,103],"platform":"nix","media_mode":"desktop","platform_name":"","grade":"A"};


    var TemplGlobals = {"image_siteroot":"http://f4.bcbits.com","siteroot":"http://bandcamp.com","static_siteroot":"https://s4.bcbits.com","is_https":true,"image_formats":[{"resize_algo":"original","name":"original","id":0,"file_format":null},{"resize_algo":"scrub","name":"fullsize","id":1,"file_format":"original"},{"width":350,"resize_algo":"thumb","name":"art_thumb","height":350,"id":2,"file_format":"JPEG"},{"width":100,"resize_algo":"thumb","name":"art_thumbthumb","height":100,"id":3,"file_format":"JPEG"},{"width":300,"resize_algo":"thumb","name":"art_embedded_metadata","height":300,"id":4,"file_format":"JPEG"},{"width":700,"resize_algo":"thumb","name":"art_embedded_metadata_large","height":700,"id":5,"file_format":"JPEG"},{"width":100,"resize_algo":"thumb","name":"art_embedded_player","height":100,"id":6,"file_format":"JPEG"},{"width":150,"resize_algo":"thumb","name":"art_embedded_player_large","height":150,"id":7,"file_format":"JPEG"},{"width":124,"resize_algo":"thumb","name":"art_tags","height":124,"id":8,"file_format":"JPEG"},{"width":210,"resize_algo":"thumb","name":"art_tags_large","height":210,"id":9,"file_format":"JPEG"},{"width":1200,"resize_algo":"fit","name":"screen","height":1200,"id":10,"file_format":"JPEG"},{"width":172,"resize_algo":"thumb","name":"art_tag_search","height":172,"id":11,"file_format":"JPEG"},{"width":138,"resize_algo":"thumb","name":"art_artist_index","height":138,"id":12,"file_format":"JPEG"},{"width":380,"resize_algo":"thumb","name":"art_solo_feature","height":380,"id":13,"file_format":"JPEG"},{"width":368,"resize_algo":"thumb","name":"art_feature","height":368,"id":14,"file_format":"JPEG"},{"width":135,"resize_algo":"thumb","name":"art_feed_new_release","height":135,"id":15,"file_format":"JPEG"},{"quality":70,"width":700,"minsize":{"format":5,"size":30000},"resize_algo":"thumb","name":"art_app_large","height":700,"id":16,"file_format":"JPEG"},{"width":1024,"resize_algo":"fit","name":"bio_screen","height":1024,"id":20,"file_format":"JPEG"},{"width":120,"resize_algo":"fit","name":"bio_thumb","height":180,"id":21,"file_format":"JPEG"},{"width":25,"resize_algo":"thumb","name":"bio_navbar","height":25,"id":22,"file_format":"JPEG"},{"width":300,"resize_algo":"fit","name":"bio_phone","height":300,"id":23,"file_format":"JPEG"},{"width":300,"resize_algo":"thumb","name":"bio_licensing","height":300,"id":24,"file_format":"JPEG"},{"quality":70,"width":700,"resize_algo":"fit","name":"bio_app","height":700,"id":25,"file_format":"JPEG"},{"width":800,"resize_algo":"thumb","name":"bio_subscribe","height":600,"id":26,"file_format":"JPEG"},{"width":715,"resize_algo":"thumb","name":"bio_subscribe2","height":402,"id":27,"file_format":"JPEG"},{"width":768,"resize_algo":"thumb","name":"bio_featured","height":432,"id":28,"file_format":"JPEG"},{"width":100,"resize_algo":"thumb","name":"bio_autocomplete","height":75,"id":29,"file_format":"JPEG"},{"width":1024,"resize_algo":"fit","name":"package_screen","height":1024,"id":31,"file_format":"original"},{"width":380,"resize_algo":"thumb","name":"package_solo_feature","height":285,"id":32,"file_format":"JPEG"},{"width":368,"resize_algo":"thumb","name":"package_feature","height":276,"id":33,"file_format":"JPEG"},{"width":400,"resize_algo":"thumb","name":"package_page","height":300,"id":36,"file_format":"JPEG"},{"width":168,"resize_algo":"thumb","name":"package_thumb","height":126,"id":37,"file_format":"JPEG"},{"width":144,"resize_algo":"thumb","name":"package_thumb_small","height":108,"id":38,"file_format":"JPEG"},{"width":210,"resize_algo":"thumb","name":"fan_bio_thumb","height":210,"id":41,"file_format":"JPEG"},{"width":50,"resize_algo":"thumb","name":"fan_bio_thumb_small","height":50,"id":42,"file_format":"JPEG"},{"width":99999,"resize_algo":"fit","name":"fan_banner","height":100,"id":43,"file_format":"JPEG"},{"width":140,"resize_algo":"thumb","name":"results_grid","height":140,"id":50,"file_format":"JPEG"},{"width":975,"max_height":180,"resize_algo":"thumb","name":"custom_header_desktop","allow_transparency":true,"id":100,"file_format":"original"},{"width":750,"resize_algo":"fit","name":"custom_header_paypal","height":90,"id":101,"file_format":"JPEG"},{"width":640,"max_height":124,"resize_algo":"thumb","name":"custom_header_phone","id":120,"file_format":"JPEG"},{"resize_algo":"scrub","name":"design_background","id":130,"file_format":"JPEG"},{"width":60,"resize_algo":"thumb","name":"subscribe_message","height":45,"id":140,"file_format":"JPEG"},{"width":1280,"resize_algo":"thumb","name":"video_landscape","height":720,"id":150,"file_format":"JPEG"},{"width":720,"resize_algo":"thumb","name":"video_portrait","height":1280,"id":151,"file_format":"JPEG"},{"top":0,"width":60,"resize_algo":"thumb_crop","name":"buy_full_email_thumb_montage","height":100,"id":160,"file_format":"JPEG","left":40},{"width":750,"resize_algo":"fit","name":"weekly_mobile_web","height":422,"id":170,"file_format":"JPEG"}],"base_port_str":null,"sitedomain":"bandcamp.com","siteroot_https":"https://bandcamp.com","siteroot_current":"https://bandcamp.com","image_siteroot_https":"https://f4.bcbits.com","custom_domains_active":true};

    var JSLoader = {
        currentset : { foot: [] },

        writeScriptTags: function( section ) {
            var libs = JSLoader.currentset[ section ];
            if ( !libs )
                throw "couldn't find js section: " + section;
            for ( var i=0; i < libs.length; i++ ) {
                document.write( '<script type="text/javascript" src="' + libs[i] + '"><\/s' + 'cript>' );
            }
        },

        requireJS : function(src) {
            if(typeof(src) == "string")
            {
                JSLoader._addRequire(src);
            }
            else
            {
                for(var i=0; i<src.length; i++)
                {
                    JSLoader._addRequire(src[i]);
                }
            }
        },

        _addRequire : function(src) {
            var libs = JSLoader.currentset.foot;
            for(var i=0; i<libs.length; i++)
            {
                if(libs[i] == src) return;
            }
            libs.push( src );
        }
    };

</script>

    <script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/global_head_bundle_min_606f320055960becc72c6fd4940bb3e1.js"></script>




    <script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/tralbum_head_bundle_min_975781234d75da6247df171e9f3b00a9.js"></script>




    <script type="text/javascript">
        (function(){
            function snickerdoodle(which) {
                var pairs = document.cookie.split(/;\s*/);
                for (var i = 0; i < pairs.length; ++i) {
                    var pair = pairs[i].split("=");
                    if (pair[0] == which) {
                        return pair[1];
                    }
                }
                return null;
            }


            // pull id out of the cookies
            window.ClientID = snickerdoodle("client_id");

        })();
    </script>


    <script type="text/javascript">
            if ( window.MediaView ) {

                MediaView.initServer("desktop");

            }

            var gCrumb = null;
            var _crumbs = {};
            var UploadInfo = null;

            Control.registerEvents();
            Control.registerController( "/no_js", TPController );

        </script>










    <!--@@PAGEBUCKETS@@-->
</head>



<body class="webkit ">

<div id="pagedata" data-blob="{&quot;recaptcha_public_key&quot;:&quot;6LfhSPgSAAAAAPwto_qzHuwSmjgfrkg35xXXu_8K&quot;,&quot;lo_querystr&quot;:&quot;?action_sig=20b09ffc79d0cc854c19e9d556b3639d&amp;action_url=https%3A%2F%2Fturboinferno.bandcamp.com%2F&amp;band_id=3501376768&amp;item_id=&amp;item_type=&quot;,&quot;shows_list&quot;:null,&quot;cfg&quot;:{&quot;no_flash_uploads&quot;:true},&quot;templglobals&quot;:{&quot;is_phone&quot;:false,&quot;endpoint_mobilized&quot;:true}}"></div>











<div class="bannercontainer">








</div>









<div id="centerWrapper" >
    <div id="propOpenWrapper">












        <div id="pgBd" class="yui-skin-sam">



























            <style type="text/css">

                #propOpenWrapper {
                background: #FFFFFF;
                }

                #pgBd {
                background: #FFFFFF;
                color: #737373;
                }

                a.custom-color, #trackInfo a, #trackInfo .buy-link, #tagArea a, #rightColumn a, #name-section a, #indexpage a, .editable-grid a, .featured-grid a, #band-navbar a, #merch-item p a, #merch-item .buy a, #merch-item .buy button, #merch-item .more-merch a, .share-collect-controls a, .share-collect-controls button, .follow-unfollow, .follow-unfollow div, .collected-by a, .subscribe a, .sub a, .video-list a, .sub .main a, .subwelcome a, .artists-grid-name a, .featured-grid-name, .label-welcome .buttons a.new, .themed .label-band-selector a.themeable {
                color: #B12027;
                }

                .themed .label-band-selector .bands-menu {
                border: 1px solid #FFFFFF;
                }

                .themed .label-band-selector .bands-menu,
                .themed .label-band-selector .bands-menu .menu-artistpic {
                background: #FFFFFF;
                }

                .label-welcome h2, .label-welcome p {
                color: #737373;
                }

                .label-welcome .buttons a.new {
                box-shadow: 2px 2px #B12027;
                }

                .label-welcome .buttons a.existing {
                box-shadow: 2px 2px #737373;
                }

                .primaryText,
                #discography a.primaryText,
                #trackInfo a.title_link,
                .label-welcome .buttons a.existing {
                color: #737373;
                }

                .secondaryText {
                color: #666666;
                }

                #band-navbar a:hover {
                border-color: #B12027;
                }

                #band-navbar a.active {
                color: #737373;
                border-color: #737373;
                }

                .themed .label-band-selector .label {
                color: #666666;
                }

                .themed .label-band-selector .tabs > li {
                border-color: #666666;
                }

                .themed .label-band-selector a.selected {
                color: #737373;
                }

                .themed .label-band-selector .caret {
                fill: #B12027;
                }

                .themed .label-band-selector .selected .caret {
                fill: #737373;
                }

                #trackInfo a.notable {
                color: red; /* overrides above */
                }


                #band-navbar {
                background-color: #E6E6E6;
                }


                body {
                background: #FFFFFF;

                background-position: left top;
                background-repeat: repeat;

                background-image: none;

                }



                body.has-menubar {
                background-position: left 48px;
                }
                body.has-top-banner {
                background-position: left 40px;
                }
                body.has-menubar.has-top-banner {
                background-position: left 88px;
                }


                #pgHd, #pgFt {
                background-color: #BFBFBF;
                }

                .just-icon {
                color: #FFFFFF;
                }

                .just-icon-bg {
                border-right-color: #737373;
                }

                .collected-by .ellipsis-text {
                color: #B12027;
                }
                .collected-by .ellipsis-bg {
                background-color: #B12027;
                }

                button.subtop,
                button.subbot {
                background: #B12027;
                color: #FFFFFF;
                }

                .sub .includeditems .item a {
                color: #737373;
                }

                .sub .post-message-link {
                background-color: rgba(177, 32, 39, 0.2);
                color: #B12027;
                }

                .sub ul.tierslist li.tier {
                border-color: rgba(177, 32, 39, 0.2);
                }

                .sub button.tierbutton .spacer {
                background-color: #FFFFFF;
                }

                ol.sub-navbar li.active,
                .invertIconography ol.sub-navbar li.active {
                background-color: #FFFFFF;
                }

                ol.sub-navbar a {
                color: #B12027;
                }

                .post-subscribe .button-background {
                background-color: rgba(177, 32, 39, 0.2);
                color: #B12027;
                }

                .post-subscribe .band-link-color,
                .post-subscribe .message-text button,
                .post-subscribe .new-feed-stories button,
                .post-subscribe .new-feed-stories-floater button {
                color: #B12027;
                }

                .post-subscribe .new-feed-stories-floater button {
                background-color: #FFFFFF;
                }

                .buyFullDiscography .art .art-thumb {
                background-color: #FFFFFF;
                }

                .collected-by .loading {
                color: #737373;
                }

            </style>




            <script type="text/javascript">





    var FarbtasticUrls = {
        js:  "https://s4.bcbits.com/tmpdata/cache/farbtastic_min_10f020ec2ca233fa2c6ce4361e7313b1.js",
        css: "https://s4.bcbits.com/tmpdata/cache/farbtastic_480a6f23f1e63d1f16bca062c24752f5.css"
    };
</script>





            <script type="text/javascript">



var BandData = {"has_policies":null,"fan_email":null,"can_enable_buy_full_discography":false,"has_public_tralbums":true,"local_url":"http://turboinferno.bandcamp.com","navbar_items":[{"nav_type":"m","url":"/music","title":"music"},{"nav_type":"p","url":"/merch","title":"merch"},{"nav_type":"v","url":"/video","title":"video"}],"is_label":false,"buy_full_discography_banner_dismissed":null,"has_public_merch":false,"currency":"EUR","header_desktop":{"area_tags":null,"image":{"width":975,"format":100,"image_id":5192951,"height":180,"url":"http://f4.bcbits.com/img/0005192951_100.png","https_url":"https://f4.bcbits.com/img/0005192951_100.png"},"width":4752,"image_id":5192951,"paypal_image":{"width":487,"format":101,"image_id":5192951,"height":90,"url":"http://f4.bcbits.com/img/0005192951_101.jpg","https_url":"https://f4.bcbits.com/img/0005192951_101.jpg"},"height":878,"url":null,"using_map":null},"create_date":"11 Feb 2014 11:23:31 GMT","navbar_hide":false,"google_analytics_id":null,"url_hints":{"subdomain":"turboinferno","custom_domain":null,"custom_domain_verified":null},"genre_id":11,"has_any_downloads":true,"thanks_enabled":1,"public_video_ct":null,"https_local_url":"https://turboinferno.bandcamp.com","subdomain":"turboinferno","name":"Turbo Inferno","has_items_for_sale":true,"has_discounts":false,"has_label":false,"reviews_enabled":1,"id":3501376768,"private_video_ct":null,"has_recommendations":false,"merch_enabled":true,"paypal_matching_label_id":null,"url":"http://turboinferno.bandcamp.com","sites":[{"nav_type":null,"url":"http://www.turboinferno.com","title":"turboinferno.com"}],"has_download_codes":false,"playback_limits":{"streaming_limits_enabled":true,"streaming_limit":3},"has_tralbums":true,"design":{"bg_fixed":null,"defaultbg":false,"invert_iconography":null,"bg_behavior":"r","text_color":"737373","bg_file_id":null,"tile_bg":null,"bg_color":"FFFFFF","body_color":"FFFFFF","secondary_text_color":"666666","bg_halign":"l","link_color":"B12027","bg_image_id":null,"hd_ft_color":"BFBFBF","bg_file_name":null,"bg_src":null,"navbar_bg_color":"E6E6E6"},"header_phone":null,"https_url":"https://turboinferno.bandcamp.com"},
    ReferrerToken = "nilZ0i3501376768x1466617441",
    CurrencyData = null,
    PackagesData = null,
    TralbumData = {
        url: "http://turboinferno.bandcamp.com" + '/music'
    };

$(document).ready(function() {
    if ( window.Cart ) {
        Cart.startup();
    }
    if ( window.Showography ){
        Showography.domReady( false );
    }


    if ( window.BandFollow )
        BandFollow.init({"tralbum_id":null,"tralbum_type":null});



});

</script>



            <div id="customHeaderWrapper">
















                <div id="customHeader">

                    <div class="desktop-header">
                        <a href="http://turboinferno.bandcamp.com"><img src="https://f4.bcbits.com/img/0005192951_100.png" width="975" height="180"></a>

                    </div>


                </div>














                <div class="bannercontainer">








                </div>

            </div>



            <div class="leftMiddleColumns">















                <ol class="editable-grid music-grid columns-2   public" data-edit-callback="/music_reorder" data-initial-values="[{&quot;is_purchasable&quot;:true,&quot;artist&quot;:null,&quot;art_id&quot;:4062247239,&quot;page_url&quot;:&quot;/album/aspirante-drone-ep&quot;,&quot;featured_date&quot;:null,&quot;filtered&quot;:null,&quot;has_audio&quot;:null,&quot;subscriber_only&quot;:null,&quot;private&quot;:null,&quot;band_name&quot;:&quot;Turbo Inferno&quot;,&quot;id&quot;:22203254,&quot;invited_item&quot;:false,&quot;release_date&quot;:&quot;09 Dec 2015 00:00:00 GMT&quot;,&quot;publish_date&quot;:&quot;09 Dec 2015 17:31:51 GMT&quot;,&quot;hidden_band&quot;:null,&quot;band_id&quot;:3501376768,&quot;title&quot;:&quot;Aspirante Drone EP&quot;,&quot;type&quot;:&quot;album&quot;,&quot;pending_transfer&quot;:null},{&quot;is_purchasable&quot;:true,&quot;artist&quot;:null,&quot;art_id&quot;:2688497523,&quot;page_url&quot;:&quot;/album/demo-tape&quot;,&quot;featured_date&quot;:null,&quot;filtered&quot;:null,&quot;has_audio&quot;:null,&quot;subscriber_only&quot;:null,&quot;private&quot;:null,&quot;band_name&quot;:&quot;Turbo Inferno&quot;,&quot;id&quot;:112427749,&quot;invited_item&quot;:false,&quot;release_date&quot;:&quot;02 Jun 2015 00:00:00 GMT&quot;,&quot;publish_date&quot;:&quot;02 Jun 2015 09:46:50 GMT&quot;,&quot;hidden_band&quot;:null,&quot;band_id&quot;:3501376768,&quot;title&quot;:&quot;Demo Tape&quot;,&quot;type&quot;:&quot;album&quot;,&quot;pending_transfer&quot;:null}]">


                    <li data-item-id="album-22203254"
                        data-band-id="3501376768"
                        class="music-grid-item square first-four

    "
                        data-bind="css: {'featured': featured()}"

                    >
                        <a href="/album/aspirante-drone-ep">

                            <div class="art">


                                <img src="https://f4.bcbits.com/img/a4062247239_2.jpg" alt="" />




                            </div>

                            <p class="title">
                                Aspirante Drone EP

                            </p>

                        </a>
                    </li>

                    <li data-item-id="album-112427749"
                        data-band-id="3501376768"
                        class="music-grid-item square first-four

    "
                        data-bind="css: {'featured': featured()}"

                    >
                        <a href="/album/demo-tape">

                            <div class="art">


                                <img src="https://f4.bcbits.com/img/a2688497523_2.jpg" alt="" />




                            </div>

                            <p class="title">
                                Demo Tape

                            </p>

                        </a>
                    </li>


                </ol>




            </div>

            <div id="rightColumn" class="rightColumn music-page "
                 itemscope itemtype="http://schema.org/MusicGroup">

                <script type="text/javascript">if (Cart) { Cart.contentsScriptLoad(); }</script>



                <meta itemprop="name" content=""/>







                <div id="bio-container" data-bind="css: {'ko-ready': $data}">
                    <h3 class="title bio-label hiddenAccess">about</h3>



                    <div class="artists-bio-pic artists-bio-pic-uploaded  landscape">
                        <div class="signed-out bio-pic">
                            <a class="popupImage" href="https://f4.bcbits.com/img/0005196872_10.jpg" data-image-size="1200,1200" data-image-preload="true">

                                <img src="https://f4.bcbits.com/img/0005196872_21.jpg" class="band-photo" alt="Turbo Inferno image">
                            </a>
                        </div>
                    </div>



                    <p id="band-name-location">
                        <span class="title">Turbo Inferno</span>
                        <span class="location secondaryText">Madrid De Los Trillos, Spain</span>
                    </p>

                    <div id="following-actions">

                        <button type="button" class="follow-unfollow compound-button" style="visibility: hidden;">
                            <div>placeholder</div>
                        </button>
                    </div>


                    <div class="signed-out-artists-bio-text">

                        <meta itemprop="description" content="Bastard Noise desde la rendija de una puerta mal cerrada en la cabina de control de una estacion de metro."/>

                        <p id="bio-text">Bastard Noise desde la rendija de una puerta mal cerrada en la cabina de control de una estacion de metro.</p>


                    </div>



                    <ol id="band-links">

                        <li><a target="_blank" href="http://www.turboinferno.com" itemprop="url">turboinferno.com</a></li>

                    </ol>







                </div>









                <h3 id="contact-help" class="title">contact / help</h3>




                <p><a href="/contact?b=3501376768&amp;n=Turbo%20Inferno" title="Send an email to Turbo Inferno">Contact Turbo Inferno</a></p>



                <p><a href="/help/downloading?from=tralbum_downloading" target="_blank">Streaming and <span id="sidebar-contact-label-break"><br></span>Download help</a></p>









            </div>


            <div style="clear:both"></div>
        </div>

        <div id="pgFt">


            <div id="pgFt-inner" class="pgft-desktop">

                <div id="footer-logo-wrapper">

                    <a id="footer-logo" href="https://bandcamp.com?from=logo" ><span class="hiddenAccess">Bandcamp</span></a>

                    <span id="webapp-selector-ui" class="webapp-selector-ui" style="display:none" data-webapps="[{&quot;hostnameRange&quot;:[1,12],&quot;appNumbers&quot;:[1,7],&quot;hostnameBase&quot;:&quot;bender&quot;}]"></span>
                </div>

                <ul id="legal" class="horizNav">

                    <li id="login"><a href="https://bandcamp.com/login">log in</a></li>

                    <li><a href="https://bandcamp.com/terms_of_use">terms of use</a></li>
                    <li><a href="https://bandcamp.com/privacy">privacy</a></li>
                    <li><a href="https://bandcamp.com/copyright">copyright policy</a></li>

                    <li id="site-status"><a href="http://twitter.com/bandcampstatus" target="_blank">status</a></li>



                    <li class="view-switcher desktop"><a>switch to mobile view</a></li>


                </ul>
                <span class="static-content">&nbsp;</span>

            </div>



        </div>



    </div>
</div>





<script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/global_foot1_bundle_min_c5ea150258dc4cef4f6a5bf5af88ed8a.js"></script>
<script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/global_foot2_bundle_min_f68b8aef41ede041fe53d17a34986e04.js"></script>




<script type="text/javascript">

    JSLoader.writeScriptTags( "foot" );

</script>



<script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/jquery_lazyload_bundle_min_11a33a2b747be85cdc50e9ffa50b05e3.js"></script>
<script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/tralbum_bundle_min_181490d8de212ed9f90a89b70ae704dc.js"></script>
<script type="text/javascript" src="https://s4.bcbits.com/tmpdata/cache/tralbum_templates_bundle_430f06f31b2ad9b861cc55efe6b0359a.js"></script>

<script type="text/javascript">



    Dom.initActiveElement();
</script>



<script type="text/javascript">
    (function() {

        Y.util.Event.onDOMReady( function() {
            Control.handleDomReady();
            FieldHints.init();
            Dom.scrollToParamElt();
        });

    })();

</script>




<script type="text/javascript">
if (!isDebug) {
    var _gaq = _gaq || [];
    var _qevents = _qevents || [];


    (function() {
        var doc = document, isHttps = ('https:' == doc.location.protocol);
        function loadScript(src) {
            var n = doc.createElement('script'), s = doc.getElementsByTagName('script')[0];
            n.type = 'text/javascript'; n.async = true; n.src = src;
            s.parentNode.insertBefore(n, s);
        }


        _gaq.push(
            ['_setAccount', 'UA-4189134-2'],
            ['_setDomainName', 'bandcamp.com'],
            ['_setAllowAnchor', true],
            ['_trackPageview']

        );
        loadScript((isHttps ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js');


        _qevents.push({qacct:'p-0207-dhLCDgvg'});
        loadScript((isHttps ? 'https://secure' : 'http://edge') + '.quantserve.com/quant.js');



    })();
}
</script>
<noscript>
    <div style="display:none;"><img src="//pixel.quantserve.com/pixel/p-0207-dhLCDgvg.gif" border="0" height="1" width="1" alt=""></div>
</noscript>



</body>
</html>
<!-- bender05-6 Wed Jun 22 17:44:01 UTC 2016 -->

