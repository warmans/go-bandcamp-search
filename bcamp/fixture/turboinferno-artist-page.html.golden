{"Bio":"Bastard Noise desde la rendija de una puerta mal cerrada en la cabina de control de una estacion de metro.","Links":[{"URI":"http://www.turboinferno.com","Text":"turboinferno.com"}],"Embed":""}
